### DISCUSSION TITLE
feat: ditch cp for rsync

### DISCUSSION THREAD

**Discussion by janleigh** [janleigh/dotfiles#20] [General] [answered]
<img src="https://cdn.xndr.tech/u/sIAt0J4.png">

**Comment by aishenreemo** [comment]
issues where

**Reply by janleigh** [reply]
now on discussions

**Comment by janleigh** [comment, marked as answer]
Should be added on [8191af2](https://github.com/janleigh/dotfiles/commit/8191af2126371048821df4fce99a4449f69b6c1a)
